middleware/deserialize-user.go

```diff
@@ -0,0 +1,57 @@
+package middleware
+
+import (
+	"fmt"
+	"strings"
+
+	"tiny-site-backend/initializers"
+	"tiny-site-backend/models"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/golang-jwt/jwt"
+)
+
+func DeserializeUser(c *fiber.Ctx) error {
```

Comment:
here initializers.DB.First() method is used to retrieve the user from the database. The initializers.DB.First() method takes a where clause as an argument, and this clause is vulnerable to SQL injection attacks.

Reply:
result := initializers.DB.First(&user, "id = ?", userI